msteams-platform/concepts/design/map-use-cases.md

```diff
@@ -1,72 +1,105 @@
 
-## Create, share and collaborate on items in an external system
+> [!NOTE]
+> You must have a good understanding of the [entry points and UI elements](../../concepts/extensibility-points.md) available for your app. You must also make sure that you [considered your use cases](../../concepts/design/understand-use-cases.md) carefully.
 
-App for Microsoft Teams are a great way to interact with your data, and there are a variety of integration points to choose from.
+## Choose the correct scope for your app
 
-* Messaging extensions with search commands - Search external systems and share the results as an interactive card.
+Before choosing the app scope, consider the following:
 
-* Messaging extensions with action commands - Collect information to insert into a data store or perform advanced searches.
+* An app can exist across scopes.
+* App capabilities, such as messaging extensions, follow users across scopes.
+* Users are often hesitant to add apps to Teams or channels.
+* Guest users can access content exposed in Teams or channels.
 
-* Tabs - Create embedded web experiences to view, work with, and share data.
+You can choose between personal scope and team or channel scope for your app depending on the following:
 
-* Connectors and webhooks - A simple way to push data into, and send data out of the Teams client.
+* For selecting personal scope, answer questions, such as:
+  * Are there one-on-one interactions with the app required for privacy or other reasons? For example, checking leave balance.
+  * Is there going to be collaboration between users who might not have any common Teams? For example, finding upcoming events in a company.
+  * Are there any personal notifications or messages that will need to be sent to a user throughout the Teams app experience? For example, reminders for approvals or registrations.
 
-* Task modules - Interactive modal forms from wherever you need them to collect or display information.
+* For selecting team or channel scope, answer questions, such as: 
+  * Is the information presented by the app, either in tab or through a bot, relevant and useful for most members in a Team? For example, Scrum app.
+  * Could the app’s context change depending on the team in which it is added to? For example, Planner’s tasks are different in different teams. 
+  * Is it possible that all members in a persona who need to collaborate are a part of a single team? For example, agents working on a ticket.
+
+The following scenarios will guide you in understanding the selection of extensibility points and UI elements that work well with Teams app capabilities:
```

Comment:
we have 3rd term here - extensibility points. 

It might be a good idea to choose between the 3 terms - Extension points, extensibility points, entry points and use that across docs